src/components/NewTaxonomy/NewTaxonomy.tsx

```diff
@@ -48,12 +49,23 @@ type TaxonomyProps = {
 };
 
 const convert = (items: TaxonomyItem[], options: TaxonomyOptions): AntTaxonomyItem[] => {
-  return items.map(item => ({
-    title: item.hint ? (
+  const enrich = (item: TaxonomyItem) => {
+    // @todo marginLeft: -4 is good to align labels, but cats them in selected list
+    const color = (item: TaxonomyItem) => (
+      <span style={{ background: item.color, padding: '2px 4px', borderRadius: 2 }}>{item.label}</span>
```

Comment:
can we use BEM and css (.styl) instead of inline styles? it would be more consistent with the rest of the codebase